dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -955,9 +955,12 @@ def _test_logged_in_milestone_with_image(async_activity_writes:)
   end
 
   test 'sharing when gatekeeper has disabled sharing does not work' do
-    Gatekeeper.set('sharingEnabled', where: {script_name: @script.name}, value: false)
+    Gatekeeper.set('shareEnabled', where: {script_name: @script.name}, value: false)
 
-    response = post :milestone, @milestone_params
+    post :milestone, @milestone_params.merge(program: studio_program_with_text('hey some text'))
+
+    assert_response :success
+    response = JSON.parse(@response.body);
```

Comment:
can you also add a smilar test that does "sharing when gatekeeper has not disabled sharing does work"? The actual default sharing functionality should be covered by other tests (should != is of course), but this would make it more obvious that your proving-a-negative test is testing the right thing.